src/test/java/de/retest/recheck/ui/PathTest.java

```diff
@@ -20,6 +17,12 @@ public void string_representation_of_simple_path() throws Exception {
 		assertThat( Path.fromString( "elem[1]" ) ).isEqualTo( Path.path( ELEMENT_0 ) );
 	}
 
+	@Test
+	public void ommitted_suffix() throws Exception {
```

Comment:
Maybe more descriptive:

```suggestion
	public void should_add_missing_suffixes() throws Exception {
```